webapp/channels/src/components/channel_view/index.ts

```diff
@@ -51,13 +52,16 @@ function mapStateToProps(state: GlobalState) {
         teamUrl: getCurrentRelativeTeamUrl(state),
         isFirstAdmin: isFirstAdmin(state),
         enableWebSocketEventScope,
+        canRestrictDirectMessage: config.RestrictDirectMessage === 'team' && (channel?.type === 'D' || channel?.type === 'G'),
```

Comment:
Same here, can we rename `canRestrictDirectMessage` to `canRestrictDirectOrGroupMessage` to be consistent with backend changes?